packages/admin/cms-admin/src/pages/pageTree/PageLabel.tsx

```diff
@@ -28,12 +33,25 @@ const PageLabel: React.FunctionComponent<PageLabelProps> = ({ page, disabled, on
                     />
                 )}
             </LinkText>
+
+            {documentType?.infoTag && page.document?.__typename && (
+                <InfoPanel>{documentType?.infoTag({ __typename: page.document.__typename, ...page.document })}</InfoPanel>
+            )}
         </Root>
     );
 };
 
 export default PageLabel;
 
+const InfoPanel = styled("div")`
```

Comment:
Could we use MUI's Chip component instead? @jamesricky recently adapted their styling to fit in the COMET style

Reply:
changed it. using the chip component now.
it still needed some custom stylings to fit the design. (padding/height)

![image](https://user-images.githubusercontent.com/44967610/176919588-4381deb9-6817-400b-962c-35a4148b5c78.png)